man/aq_locations.Rd

```diff
@@ -15,6 +15,7 @@ aq_locations(
   longitude = NULL,
   radius = NULL,
   page = NULL
+  siteType = NULL
```

Comment:
There's a missing comma (but `devtools::document()` needs to be run anyway)

Reply:
oh of course! But I'm really sorry. I think I'm  a little confused with this and should maybe read more about the `devtools`-package before diving deeper into this.